### DISCUSSION TITLE
Preview comments aren't working with custom Domains

### DISCUSSION THREAD

**Discussion by emiliosheinz** [vercel/community#1202] [Help] [answered]
I have a custom preview domain for my `develop` branch as shown below:

![image](https://user-images.githubusercontent.com/103655828/210263485-29765d14-a8e7-429e-99ad-4186c6918bf5.png)

But, unfortunately when I access this custom domain the preview comments don't work. 

Just so you know, when I access the Automatic URL generated by Vercel during the Deployments everything works.

![image](https://user-images.githubusercontent.com/103655828/210263807-8a0c36b6-d523-4074-8da8-5a0bc50fd069.png)

Do you happen to know if this is expected or if there is something that I need to do to get it to work?

I have already tried accessing it from different devices, in incognito mode, and with no cache, but nothing helped.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @emiliosheinz! This is expected behavior at the moment. Preview Comments are currently only available on the deployment's [automatic generated url](https://vercel.com/docs/concepts/deployments/generated-urls) and [Preview Deployment Suffix](https://vercel.com/docs/concepts/deployments/generated-urls#preview-deployment-suffix) links. 

A problem with Preview Comments on Custom Domains was discovered at the end of December, so the feature is temporarily disabled for that scenario. We have a ticket to address the source of the problem and the engineers are working on it as they return from holiday travel. 🙇

**Reply by emiliosheinz** [reply]
Got it! Thank you for the quick response 😊

**Reply by IGassmann** [reply]
@amyegan is this still the case? I'm having some issues with our custom domain.

**Reply by amyegan** [reply]
Hi @IGassmann! To [use comments with a custom alias domain](https://vercel.com/docs/concepts/deployments/comments/specialized-usage#using-comments-with-a-custom-alias-domain), you need to explicitly [enable comments for your project](https://vercel.com/docs/concepts/deployments/comments/how-comments-work#at-the-project-level)
